src/components/JobsTrainingContent/index.jsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState } from "react";
+import { Container, Row, Col } from "react-bootstrap";
+import "./index.scss";
+import { useTranslation } from "react-i18next";
+import OpportunitiesCard from "../OpportunitiesCard";
+import SearchBar from "../SearchBar";
+import JobsTrainingButtons from "../JobsTrainingButtons";
+
+const JobsTrainingContent = () => {
+  const { t } = useTranslation();
+  const [cardsArr, setCardsArr] = useState([
+    ...t(`home.jobOpportunities`, { returnObjects: true }),
+  ]);
+
+  const [searchCard, setSearchCards] = useState(cardsArr);
```

Comment:
is a search card a search result or a card with a search on it? if the former, it needs a better name